rosetta/locale/az/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,211 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-11-01 18:01+0400\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: rosetta/templates/rosetta/admin_index.html:14
+msgid "Translations"
+msgstr "Tərcümələr"
+
+#: rosetta/templates/rosetta/admin_index.html:15
+msgid "Change"
+msgstr "Dəyiş"
+
+#: rosetta/templates/rosetta/file-list.html:4
+#: rosetta/templates/rosetta/file-list.html:7
+msgid "Language selection"
+msgstr "Dil seçimi"
+
+#: rosetta/templates/rosetta/file-list.html:7
+#: rosetta/templates/rosetta/form.html:21
+msgid "Home"
+msgstr "Ev"
+
+#: rosetta/templates/rosetta/file-list.html:8
+msgid ""
+"Couldn't load the specified language file. This usually happens when using "
+"the Encrypted Cookies Session Storage backend on Django 1.4 or higher.<br/"
+">Setting ROSETTA_STORAGE_CLASS = 'rosetta.storage.CacheRosettaStorage' in "
+"your settings file should fix this."
+msgstr ""
+
+#: rosetta/templates/rosetta/file-list.html:14
+msgid "Filter"
+msgstr "Filtr"
+
+#: rosetta/templates/rosetta/file-list.html:15
+msgid "Project"
+msgstr "Proyekt"
```

Comment:
We have "layihə" in Azerbaijani for this word. "Proyekt" is not a common (ümumişlək) word